.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ parameters:
 
   node-version:
     type: string
-    default: 16.17-browsers
+    default: 18.11-browsers
```

Comment:
The latest version is 188.12 but the cimg image doesn't seem to be available yet